src/classes/UTIL_Http.cls

```diff
@@ -54,21 +63,23 @@ public without sharing class UTIL_Http {
     public class RequestService {
 
         /***
-        * @description Sends an HttpRequest 
-        * Returned response can be a success or an error response
+        * @description Sends an HttpRequest
         * @param request HttpRequest
         * @return Response HttpResponse wrapper
         */
         public Response sendRequest(HttpRequest request) {
-            UTIL_Http.Response response = new UTIL_Http.Response();
+            Response response;
 
             try {
                 HttpResponse requestResponse = new Http().send(request);
+
```

Comment:
In order to test the Payments Services request timed out response, I could not test it without adding the timeout value in ms before the request is sent, ie before line 74 as `request.setTimeout(1000);`. 

The Gift Entry error handling is as expected:
<img width="1047" alt="GEPurchaseCalloutTimedOut" src="https://user-images.githubusercontent.com/28743286/95769959-88249700-0c6d-11eb-9bc1-2b10fdb0fb64.png">

The RD entry form error handling is as expected:
<img width="1288" alt="RDCommitmentCalloutTimedOut" src="https://user-images.githubusercontent.com/28743286/95770014-9bcffd80-0c6d-11eb-9e42-42955286abec.png">